ee/query-service/usage/manager.go

```diff
@@ -101,11 +101,24 @@ func (lm *Manager) UploadUsage(ctx context.Context) error {
 
 	// get usage from clickhouse
 	dbs := []string{"signoz_logs", "signoz_traces", "signoz_metrics"}
-	query := `select collector_id, exporter_id, tenant, data from %s.usage where timestamp>=$1 order by timestamp`
+	query := `
```

Comment:
Joins perform badly in clickhouse. Can we write a better query?

Reply:
Didn't find a better alternative to this yesterday so when ahead with it. Although the number of rows for the join will be very less we can still discuss about alternate approaches to make this better.